modules/apps/adaptive-media/adaptive-media-image-impl/src/main/java/com/liferay/adaptive/media/image/internal/processor/util/TiffOrientationTransformer.java

```diff
@@ -127,6 +134,21 @@ else if (tiffOrientationValue == _ORIENTATION_VALUE_ROTATE_270_CW) {
 		return renderedImage;
 	}
 
+	private static boolean _validateContentType(InputStream inputStream) {
+		if (ArrayUtil.contains(
+				_INVALID_CONTENT_TYPE,
```

Comment:
Can we call it _UNSUPPORTED_MIME_TYPES? For arrays/collections we have to use the plural in the name, and the mime type is not really invalid. Also, instead of `_validate...` can we use `isContentTypeSupported` or something like that? Someone reading the code might think that there's something wrong with JPEG support.